### PR TITLE
Payroll list my payments

### LINKED ISSUE #4: List and filter my payments

### PR DESCRIPTION
**Pull request by sgobotta**
This PR provides:

+ payments list in the My Payroll section
+ token and date filters for payments
+ Closes #4

### CODE REVIEW

File: future-apps/payroll/app/src/screens/components/SalaryTable.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      title: 'Status',
+      value: data => data.status,
+    },
+    {
+      name: 'transaction-address',
+      title: 'Transaction Address',
+      value: data => data.transactionAddress,
+      defaultValue: 'Active',
+      render: (formattedValue, rawValue, item) => (
+        <TransactionAddress>
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/screens/components/SalaryTable.js:32]: We could use aragon-ui `IdentityBadge` component here, it's the one being used in Finance app

File: future-apps/payroll/app/src/store/marshalling.js [hunk 2]
```diff
@@ -34,3 +34,25 @@ export function tokenAllocation (data) {
...
+    accountAddress: data.returnValues.employee,
+    amount: {
+      amount: data.returnValues.amount,
+      isIncoming: true, // FIXME: Assumption: all salaries are incoming - sgobotta
+      displaySign: true, // FIXME: The send payroll event should provide the displaysign option
+      token: {
+        address: data.token.address,
+        symbol: data.token.symbol
+      },
+      decimals: data.token.decimals
```
Comment (lmcorbalan) [inline future-apps/payroll/app/src/store/marshalling.js]: I would put this inside token to maintain consistency, like:
```
token: {
  address: data.token.address,
  symbol: data.token.symbol,
  decimals: data.token.decimals
},
```

Comment (sgobotta) [reply future-apps/payroll/app/src/store/marshalling.js]: Fixed the attribute

File: future-apps/payroll/app/src/screens/components/SalaryTable.js (hunk 1 above)
Comment (sgobotta) [reply future-apps/payroll/app/src/screens/components/SalaryTable.js:32]: We're not using IdentityBadge but we're planning on create a custom component using SafeLink. The approach has been pushed.

**Pull request merged**